src/merge-tools.toml

```diff
@@ -1,4 +1,6 @@
 [merge-tools.kdiff3]
+# --merge to open output pane, CreateBakFiles=0 to not include bakcup files in commit
```

Comment:
nit: typo in "backup"

Reply:
Oops. I haven't enabled flyspell in toml-mode.